PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -792,14 +687,88 @@ struct RobustFluctuationObservables {
     histosEvent.fill(HIST("hNtracksBef"), tracks.size());
 
     auto collBC = bc.globalBC() % 3564;
-    uint64_t globalFoundBC = -1;
+    uint64_t globalFoundBC = 9999;
     // int64_t globalFoundBC = -1;
     if (collision.has_foundBC()) {
       auto bcFound = collision.foundBC_as<BCsRun3>(); // collision.foundBC();
       globalFoundBC = bcFound.globalBC() % 3564;
     }
     histosEvent.fill(HIST("hBC_Bef"), collBC);
 
+    // cut ITS ROF by hand, with configurables
+    bool flagBCisNotInHandmadeBoundariesITSROF = true;
+    if (collBC > nITSROF_BC_cutWidth / 2 && collBC < (nBCsPerOrbit - nITSROF_BC_cutWidth / 2)) {
+      for (int iROF = 0; iROF < nITSROF; iROF++) {
+        int ROF_BC_center = nITSROF_BC_offset + iROF * nBCsPerOrbit / nITSROF;
+        if (fabs(static_cast<int>(collBC) - ROF_BC_center) <= nITSROF_BC_cutWidth / 2) {
+          flagBCisNotInHandmadeBoundariesITSROF = false;
+          if (TFid < 2)
+            cout << "QA: collBC = " << collBC << ", nITSROF_BC_offset = " << nITSROF_BC_offset
+                 << ", ROF_BC_center = " << ROF_BC_center << ", nITSROF_BC_cutWidth = " << nITSROF_BC_cutWidth
+                 << ", fabs(collBC - ROF_BC_center) = " << fabs(static_cast<int>(collBC) - ROF_BC_center)
+                 << ", (int)collBC - ROF_BC_center = " << static_cast<int>(collBC) - ROF_BC_center
+                 << endl;
```

Comment:
For the next iteration, please, no `cout` nor `endl`
Use instead the O2 logging mechanism `LOGF`